exercises/space-age/pubspec.yaml

```diff
@@ -0,0 +1,5 @@
+name: 'space_age'

```

Comment:
This has CRLF line-endings. Could you change it to LF?

It's better for git to store files with LF and then on the client-side you can set your config to convert it for your environment to CRLF if needed.

Reply:
Interesting, I have LF ending set, even in windows, for all of my editors. This has never been a problem. I copied pubspec from another exercise, don't remember which one, that's probably the culprit. I'll check if any other file has that.

Reply:
Do you see it too? If it's just on my machine, then we can ignore it.

Reply:
Actually, that red marker means there is no extra line at the end. And I think it's a standard to have an extra empty line ending the file. I may have used a code formatter which removed the line. It doesn't seem to be an actual problem for git. @Stargator

Reply:
Learned something new today, it seems most Unix tools expect a file to end with a new line so that when they display its contents in the console they don't have to worry about differentiating the file contents with actual log/status messages. By having a new line the tools don't add any extra line and the output is distinguishable automatically.

Reply:
@devkabiir I added a commit that changes the line ending. I wasn't referring to a red marker. Feel free to fixup my commit into yours, so it's one commit.

I took a screenshot of the change to highlight it, the left side is CRLF (Windows) and the right is LF (Mac, Linux):

<img width="1675" alt="screen shot 2018-08-24 at 8 35 11 am" src="https://user-images.githubusercontent.com/7527155/44584975-b33f8a00-a778-11e8-951a-025dc850a6b9.png">